### DISCUSSION TITLE
how to pass custom rpc url to aptos config??

### DISCUSSION THREAD

**Discussion by ajaythxkur** [aptos-labs/aptos-developer-discussions#383] [Questions]
`const aptosConfig = new AptosConfig({ fullnode: rpcUrl });
        this.aptos = new Aptos(aptosConfig);`
      I am using devnet fullnode url, the issue is that everything works fine when i do network: Network.DEVNET but if i pass url i get account not found error

**Comment by ajaythxkur** [comment]
anyone??

**Reply by 0x-j** [reply]
try this `new AptosConfig({ network: Network.TESTNET, fullnode: "your_node_url" });`

**Reply by ajaythxkur** [reply]
Well isn't it possible without network?? i am dynmaically passing mainnet, devnet, testnet urls and don't have track of them about their network, i just get the urls from somewhere
